docs/operate/reference/components/camera/_index.md

```diff
@@ -85,6 +85,28 @@ Provide at least the width and height values to start.
 
 {{% /expand%}}
 
+{{% expand "Frame read failed: Too many retries" %}}
+
+**Description:** This error typically occurs with IR camera modules and similar specialized cameras.
+It happens when the camera module is still initializing or calibrating itself, but `viam-server` is already attempting to read frames from it.
+This can also be caused by I2C communication issues between the board and the camera module, particularly on Raspberry Pi 5 systems.
+
+**Solution:** Try one of the following approaches:
+
+1. **Reduce the refresh rate for I2C cameras**:
+   For I2C-based cameras (like some IR modules), try setting the refresh rate to a lower value (for example, 2Hz instead of 4Hz).
+   This can help with I2C bus timing issues, especially on Raspberry Pi 5 systems.
+
+1. **Check physical connections**:
+
+   - Ensure the camera module is properly connected.
+   - For I2C cameras, verify that the SCK (clock) pin is properly connected and not shorted to ground or power.
+     If the pin has been shorted, this can permanently damage the hardware.
+
+If these steps don't resolve the issue, check your machine logs for additional error messages that might provide more specific information about the problem.
+
+{{% /expand%}}
+
```

Comment:
This error might not be generic to all i2c IR cameras - we only have the one module using adafruit's APIs https://github.com/randhid/waveshare-thermal.

It's also mine, not viam official. https://github.com/randhid/waveshare-thermal. Do you want me to just put this guide on that readme?